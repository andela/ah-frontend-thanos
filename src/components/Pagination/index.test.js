import React from 'react';
import { shallow } from 'enzyme';
import Pagination from '.';


describe('Test  pagination', () => {
  const article = {
    count: 35,
  };
  const pageNumber = '1';
  const Paginationwrapper = shallow(
    <Pagination article={article} pageNumber={pageNumber} />,
  );

  test('Snap shot pagination Component', () => {
    expect(Paginationwrapper).toMatchSnapshot();
  });
  test('Pagination should render correctly', () => {
    expect(Paginationwrapper.find('.active').length).toBe(1);
    expect(Paginationwrapper.find('.disabled').length).toBe(1);
  });
});
