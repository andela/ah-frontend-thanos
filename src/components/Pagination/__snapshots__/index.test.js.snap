// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test  pagination Snap shot pagination Component 1`] = `
<div>
  <br />
  <ul
    className="pagination justify-content-center"
  >
    <li
      className="page-item disabled"
    >
      <a
        className="page-link"
        href="articles/page/0"
        tabIndex="-1"
      >
        Previous
      </a>
    </li>
    <li
      className="page-item active"
      key="1"
    >
       
      <a
        className="page-link"
        href="articles/page/1"
      >
        1
      </a>
    </li>
    <li
      className="page-item null"
      key="2"
    >
       
      <a
        className="page-link"
        href="articles/page/2"
      >
        2
      </a>
    </li>
    <li
      className="page-item null"
      key="3"
    >
       
      <a
        className="page-link"
        href="articles/page/3"
      >
        3
      </a>
    </li>
    <li
      className="page-item null"
    >
      <a
        className="page-link"
        href="articles/page/2"
      >
        Next
      </a>
    </li>
  </ul>
</div>
`;
